src/OSPSuite.Presentation/Presenters/Charts/CurveMultiItemEditorPresenter.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Drawing;
+using OSPSuite.Core.Chart;
+using OSPSuite.Presentation.Views.Charts;
+
+namespace OSPSuite.Presentation.Presenters.Charts
+{
+   public enum YesNoValues
```

Comment:
@msevestre no particularly proud of this solution here....still I preferred Yes/No instead of true/false and the enum keeps things consistent with the other comboboxes.